exporter/opentelemetry-exporter-prometheus-remote-write/tests/test_prometheus_remote_write_exporter.py

```diff
@@ -0,0 +1,337 @@
+            timeseries[2], (MinMaxSumCountAggregator, "testname_sum", 6)
+        )
+        self.assertEqual(
+            timeseries[3], (MinMaxSumCountAggregator, "testname_count", 2)
+        )
+
+    # Ensures histogram aggregator is correctly converted to timeseries
+    def test_convert_from_histogram(self):
+        histogram_record = self._generate_record(HistogramAggregator)
+        histogram_record.aggregator.update(5)
+        histogram_record.aggregator.update(2)
+        histogram_record.aggregator.update(-1)
+        histogram_record.aggregator.take_checkpoint()
+
+        self._exporter.create_timeseries = self._converter_mock()
+        timeseries = self._exporter.convert_from_histogram(histogram_record)
+        self.assertEqual(
+            timeseries[0], (HistogramAggregator, 'testname_bucket{le="0"}', 1)
+        )
+        self.assertEqual(
+            timeseries[1],
+            (HistogramAggregator, 'testname_bucket{le="+Inf"}', 2),
+        )
+        self.assertEqual(
+            timeseries[2], (HistogramAggregator, "testname_count", 3)
+        )
+
+    # Ensures last value aggregator is correctly converted to timeseries
+    def test_convert_from_last_value(self):
+        last_value_record = self._generate_record(LastValueAggregator)
+        last_value_record.aggregator.update(1)
+        last_value_record.aggregator.update(5)
+        last_value_record.aggregator.take_checkpoint()
+
+        self._exporter.create_timeseries = self._converter_mock()
+        timeseries = self._exporter.convert_from_last_value(last_value_record)
+        self.assertEqual(timeseries[0], (LastValueAggregator, "testname", 5))
+
+    # Ensures value observer aggregator is correctly converted to timeseries
+    def test_convert_from_value_observer(self):
+        value_observer_record = self._generate_record(ValueObserverAggregator)
+        value_observer_record.aggregator.update(5)
+        value_observer_record.aggregator.update(1)
+        value_observer_record.aggregator.update(2)
+        value_observer_record.aggregator.take_checkpoint()
+
+        self._exporter.create_timeseries = self._converter_mock()
+        timeseries = self._exporter.convert_from_value_observer(
+            value_observer_record
+        )
+        self.assertEqual(
+            timeseries[0], (ValueObserverAggregator, "testname_min", 1)
+        )
+        self.assertEqual(
+            timeseries[1], (ValueObserverAggregator, "testname_max", 5)
+        )
+        self.assertEqual(
+            timeseries[2], (ValueObserverAggregator, "testname_sum", 8)
+        )
+        self.assertEqual(
+            timeseries[3], (ValueObserverAggregator, "testname_count", 3)
+        )
+        self.assertEqual(
+            timeseries[4], (ValueObserverAggregator, "testname_last", 2)
+        )
+
+    # Ensures quantile aggregator is correctly converted to timeseries
+    # TODO: Add test once method is implemented
+    def test_convert_from_quantile(self):
```

Comment:
I understand that this PR is part of a series of PRs, I just want to let you know that in other circumstances having tests that do nothing and always pass mess up with your testing results since Pytest considers them to be passed test cases when really nothing has been tested. I would recommend not having them at all, commenting them out or marking them with [`skip`](https://docs.pytest.org/en/latest/skipping.html#skipping-test-functions) (in that order of recommendation).

Reply:
I did not think of that. Got rid of the empty test leaving only a comment as a reminder of where to implement this test when the method is implemented.